src/lib.rs

```diff
@@ -212,6 +215,23 @@ pub enum CompoundTagError<'a> {
     },
 }
 
+impl<'a> std::error::Error for CompoundTagError<'a> {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        None
```

Comment:
Ain't here should be provided all enum variants with their variables?

Reply:
`source` is for nested errors, of which there are none.